### DISCUSSION TITLE
School not on list

### DISCUSSION THREAD

**Discussion by ghost** [community/community#17721] [GitHub Education]
Hello,
When applying for the student account, it asks to select a school from the list. It shows most of the schools in my area except mine.
The building along with its name is visible on the map but just not as an option to select.

My school is Colaiste Chu Chulainn, located in Marshes Lower, Dundalk, Co. Louth, A91 F75P, Ireland

Website: colaistecc.ie

https://www.google.com/maps/place/Col%C3%A1iste+Ch%C3%BA+Chulainn,+Marshes+Lower,+Dundalk,+Co.+Louth/@54.0003501,-6.3953514,17z/data=!4m2!3m1!1s0x4860cea1753deb37:0xd216a659e9d1450f

**Comment by mlarasusan** [comment]
Hello [👋](https://emojipedia.org/waving-hand/) If your institution doesn’t qualify you for the Pack, ask an instructor to apply for your school to become a [GitHub Campus partner](https://education.github.com/schools).
